physics/rigid_motion_body.hpp

```diff
@@ -57,8 +57,8 @@ RigidMotion<FromFrame, ToFrame>::angular_velocity_of() const {
   } else if constexpr (std::is_same_v<F, FromFrame>) {
     return Inverse().angular_velocity_of_to_frame_;
   } else {
-    static_assert(std::disjunction_v<std::is_same_v<F, ToFrame>,
-                                     std::is_same_v<F, FromFrame>>,
+    static_assert(std::disjunction_v<std::is_same<F, ToFrame>,
+                                     std::is_same<F, FromFrame>>,
```

Comment:
It is disturbing that this ever compiled, let alone on both clang and MSVC!
I think it would be more readable to have `std::is_same_v<meow> || std::is_same_v<purr>`; not sure why I didn’t flag that when I reviewed the PR that added those `disjunction`s.

Reply:
Done.